csrankings-m.csv

```diff
@@ -1499,6 +1499,7 @@ Meng Jin 0002,Shanghai Jiao Tong University,https://yume-sjtu.github.io/,CN2l9GE
 Meng Tang 0001,Univ. of California - Merced,http://www.mengtang.org,6DmW9v0AAAAJ
 Meng Wang 0002,University of Bristol,https://mengwangoxf.github.io,NwDbSggAAAAJ
 Meng Xu 0001,University of Waterloo,https://cs.uwaterloo.ca/~m285xu,NOSCHOLARPAGE
+Meng Yan 0001,Chongqing University,https://yanmeng.github.io/,UBTGNMwAAAAJ

```

Comment:
"Research Professor" - only tenure-track faculty are eligible for inclusion. Please remove.

Reply:
Done. Thank you for the kind reminder